presto-spark-base/src/test/java/com/facebook/presto/spark/PrestoSparkQueryRunner.java

```diff
@@ -204,7 +204,7 @@ public PrestoSparkQueryRunner(String defaultCatalog)
                 ImmutableMap.of(
                         "presto.version", "testversion",
                         "query.hash-partition-count", Integer.toString(NODE_COUNT * 2),
-                        "prefer-distributed-union", "false"),
+                        "prefer-distributed-union", "true"),
```

Comment:
Do you think at this point we can simply remove the `prefer-distributed-union` property? As it was added as a hack originally.

CC: @wenleix

Reply:
`prefer-distributed-union` can be removed. Let's do it later, because if this PR broke anything, we have an option to use single mode union

Reply:
Let's at least enable the `prefer-distributed-union` by default here: https://github.com/prestodb/presto/blob/master/presto-spark-base/src/main/java/com/facebook/presto/spark/PrestoSparkSettingsRequirements.java#L98

Reply:
Also remove the setting override in the integration tests: https://github.com/prestodb/presto/blob/master/presto-spark-testing/src/test/java/com/facebook/presto/spark/testing/TestPrestoSparkLauncherIntegrationSmokeTest.java#L151

Reply:
Yeah, sounds good to just remove it.